app/views/transactions/_form.html.erb

```diff
@@ -1,32 +1,34 @@
-<%= form_with(model: transaction) do |form| %>
-  <% if transaction.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(transaction.errors.count, "error") %> prohibited this transaction from being saved:</h2>
+<div class="col-6">
+  <%= form_with(model: transaction) do |form| %>
+    <% if transaction.errors.any? %>
+      <div id="error_explanation">
+        <h2><%= pluralize(transaction.errors.count, "error") %> prohibited this transaction from being saved:</h2>
 
-      <ul>
-        <% transaction.errors.each do |error| %>
-          <li><%= error.full_message %></li>
-        <% end %>
-      </ul>
-    </div>
-  <% end %>
+        <ul>
+          <% transaction.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
 
-  <div class="field">
-    <%= form.label :amount_value %>
-    <%= form.number_field :amount_value %>
-  </div>
-  <% if transaction.new_record?%>
-    <div class="field">
-      <%= form.label :person_category_id %>
-      <%= form.collection_select(:person_category_id, PersonCategory.where(person_id: @people), :id, :category, {}, {required: true})%>
+  <div class="form-group row mb-3">
+      <%= form.label :amount_value %>
+      <%= form.number_field :amount_value, class: "form-control" %>
     </div>
-  <% else %>
-      <div class="field">
-        <%= form.hidden_field :person_category_id %>
-    </div>
-  <% end%>
+    <% if transaction.new_record?%>
+  <div class="form-group row mb-3">
+        <%= form.label :person_category_id %>
+        <%= form.collection_select(:person_category_id, PersonCategory.where(person_id: @people), :id, :category, {}, {required: true, class: "form-control"})%>
+      </div>
+    <% else %>
```

Comment:
:category.name for value - doesnot work ((
Ideal I want to see "person - category". I have no idea how to deal with it

Reply:
U can use .select
documentation: https://api.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html#method-i-select

Reply:
Corrected